src/sgame/sg_cmds.cpp

```diff
@@ -4442,6 +4451,58 @@ void Cmd_PrivateMessage_f( gentity_t *ent )
 	}
 }
 
+
+/*
+=================
+Cmd_ReplyPrivateMessage_f
+
+Reply by private message to the last private message sender
+=================
+*/
+
+void Cmd_ReplyPrivateMessage_f(gentity_t *ent)
+{
+	char *msg;
+	int target;
+	bool teamonly = false;
+
+	if (!g_privateMessages.Get() && ent)
+	{
+		ADMP( "\"" N_("Sorry, but private messages have been disabled") "\"" );
+		return;
+	}
+
+	if (trap_Argc() < 2)
+	{
+	ADMP( QQ( N_("usage: /r [message]") ) );
+	return;
+	}
+
+	if ( level.time - ent->client->pers.lastPrivateMessageSenderTime < 3000 )
+	{
+		ADMP( "\"" N_("More than one possible recipient, refusing to send.") "\"" );
```

Comment:
Maybe we should also print who we would have sent the PM to?

Reply:
done